exonum-java-binding-core/src/main/java/com/exonum/binding/storage/database/ViewModificationCounter.java

```diff
@@ -31,8 +31,9 @@
  * (i.e., if each thread has its own Views, which must be the case for Views are not thread-safe).
  * Such property is useful in the integration tests.
  */
-// TODO(dt): when we migrate to RocksDB, extract the interface and implement it inside
-//           each collection.
+// TODO(dt): when we migrate to RocksDB, extract the interface and implement it inside…
```

Comment:
why `…`?

Reply:
To see from the IDE that shows only the first comment line that there is more. It's not a proper sentence though, so ellipsis is not necessary to see the sentence is not finished.

Reply:
👍